<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DispEditor</class>
 <widget class="QWidget" name="DispEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1061</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="DisplayEdit" name="displayEdit" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="colorWidget" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="1">
      <widget class="QPushButton" name="loadCButton">
       <property name="text">
        <string>Import Bin</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="2">
      <widget class="QCheckBox" name="antialiasBox">
       <property name="text">
        <string>Antialias when loading</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QPushButton" name="clearButton">
       <property name="text">
        <string>Clear Screen</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="savePngButton">
       <property name="toolTip">
        <string>Save image, as seen above, to PNG file.</string>
       </property>
       <property name="text">
        <string>Save PNG</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="2">
      <widget class="QCheckBox" name="showLayer2Box">
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>Show Layer 2</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="saveCButton">
       <property name="text">
        <string>Export Bin</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QCheckBox" name="ditherBox">
       <property name="text">
        <string>Dither when loading</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QSpinBox" name="imgScaleBox">
       <property name="prefix">
        <string>Preview Scale: </string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>10</number>
       </property>
       <property name="value">
        <number>2</number>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="formatBox">
       <item>
        <property name="text">
         <string>Indexed 2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Indexed 4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Indexed 16</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>RGB332 (TODO)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>RGB565 (TODO)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>RGB888 (TODO)</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="loadPngButton">
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>Load Image</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QScrollArea" name="scrollArea">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>262</width>
          <height>256</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="ImageWidgetDisp" name="imageWidget" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>256</width>
             <height>256</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>256</width>
             <height>256</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QPushButton" name="helpButton">
       <property name="toolTip">
        <string>Show Instructions</string>
       </property>
       <property name="text">
        <string>Help</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QPushButton" name="clearLayer2Button">
       <property name="text">
        <string>Clear Layer 2</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Format</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0" colspan="2">
      <widget class="QLabel" name="ColorNowLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>256</width>
         <height>40</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>2</number>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DisplayEdit</class>
   <extends>QWidget</extends>
   <header>display_tool/displayedit.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ImageWidgetDisp</class>
   <extends>QWidget</extends>
   <header>display_tool/imagewidgetdisp.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
